Dockerfile

```diff
@@ -1,6 +1,6 @@
 # vim: set ft=conf
 
-FROM python:2.7
+FROM python:3.7
 ENV PYTHONUNBUFFERED 1
 ENV PYTHONPATH /app
 RUN apt-get update -qq && apt-get install -y python-mysqldb default-mysql-client
```

Comment:
I needed to add  `python-dev` to packages that get installed for compiling `mysqlclient` and `psycopg2` to get the tests to run with `docker-compose run --rm web tox`

Reply:
I see. I might have already installed `python-dev` so didn't have to install it for this project.